spec/factories/claim/advocate_claims.rb

```diff
@@ -59,6 +59,16 @@
       case_type { association(:case_type, :graduated_fee) }
     end
 
+    trait :agfs_scheme_10 do
+      after(:create) do |claim|
+        claim.defendants.each do |defendant|
```

Comment:
💭 Consider making this conditional, so that, if the _developer_ defines a new _claim_ with their own **defendants** or **representation orders**, that data is not overwritten by this.